include/zephyr/net/conn_mgr_connectivity.h

```diff
@@ -26,7 +26,7 @@ extern "C" {
  * @brief Connection Manager Connectivity API
```

Comment:
For the commit `net: conn_mgr: set to unstable`, please add information to commit msg why this change is needed.

Reply:
done